pkg/resource/v3/resource.go

```
@@ -43,7 +43,7 @@ func GetHTTPConnectionManager(filter *listener.Filter) *hcm.HttpConnectionManage
 
 	// use typed config if available
 	if typedConfig := filter.GetTypedConfig(); typedConfig != nil {
-		ptypes.UnmarshalAny(typedConfig, config)
+		_ = ptypes.UnmarshalAny(typedConfig, config)
```

Why the value ignore?

---

IIRC this is errcheck again :)

---

Hm, we probably shouldn't ignore this error. Maybe we should alter the function signature and bubble up that err

---

Updated to return nil which I this is OK for this API.